addons/html_editor/static/tests/chatgpt.test.js

```diff
@@ -0,0 +1,46 @@
+import { expect, test } from "@odoo/hoot";
+import { setupEditor } from "./_helpers/editor";
+import { press, waitFor } from "@odoo/hoot-dom";
+import { animationFrame } from "@odoo/hoot-mock";
+import { contains } from "@web/../tests/web_test_helpers";
+import { insertText } from "./_helpers/user_actions";
+
+import { EXTRA_PLUGINS } from "@html_editor/plugin_sets";
+
+const PROMPT_DIALOG_TITLE = "Generate Text with AI";
+const ALTERNATIVES_DIALOG_TITLE = "AI Copywriter";
+
+test("ChatGPT dialog opens in prompt mode when selection is collapsed (with Powerbox)", async () => {
+    const { editor } = await setupEditor("<p>te[]st</p>", { config: { Plugins: EXTRA_PLUGINS } });
+
+    // Select ChatGPT in the Powerbox.
+    insertText(editor, "/");
+    insertText(editor, "ChatGPT");
```

Comment:
```suggestion
    insertText(editor, "/ChatGPT");
```

Reply:
I saw this done in another test, I assumed there was a reason for it. Good to know there wasn't!